CMakeLists.txt

```diff
@@ -71,9 +74,12 @@ endif()
 
 # ========== Install targets - 'sudo make install' ==========
 include(InstallRequiredSystemLibraries)
+set(INCLUDE_INSTALL_DIR /usr/local/include CACHE PATH "Directory in which to install system header files.")
+set(INCLUDE_INSTALL_DIR /usr/local/include CACHE PATH "Directory in which to install system libraries.")
```

Comment:
You mean `LIB_INSTALL_DIR` I guess

Reply:
I think this should fix our CI build https://travis-ci.org/SemaiCZE/inicpp/builds/482697764 . I see the compiler is awfully old, so I should update the CI setup, but this is a bit different story. I think after this fix we are ready to merge. Thanks.

Reply:
Right you are, fixed and re-tagged v1.0.1.